bin/node/src/commands/store.rs

```diff
@@ -147,15 +147,15 @@ impl StoreCommand {
         Store::bootstrap(genesis_state, data_directory)
     }
 
-    fn generate_genesis_account() -> anyhow::Result<AccountFile> {
+    pub fn generate_genesis_account() -> anyhow::Result<AccountFile> {
         let mut rng = ChaCha20Rng::from_seed(rand::random());
         let secret = SecretKey::with_rng(&mut get_rpo_random_coin(&mut rng));
 
         let (mut account, account_seed) = create_basic_fungible_faucet(
             rng.random(),
-            TokenSymbol::try_from("POL").expect("POL should be a valid token symbol"),
+            TokenSymbol::try_from("MIDEN").expect("MIDEN should be a valid token symbol"),
             12,
-            Felt::from(1_000_000u32),
+            Felt::from(u32::MAX),
```

Comment:
I think max supply should be higher and the number of decimal places smaller. Specifically, I'm thinking:

- $6$ for decimal places (down from the current 12).
- 100B in tokens or $10^{11}$. Together with 6 decimal places this will result in total supply of $10^{17}$ base units (or under $2^{57}$).

We'll probably need to update the faucet UI to issue tokens in base units - i.e., 100 tokens would translate to 100M base units.